cns/kubecontroller/nodenetworkconfig/conversion_linux_test.go

```diff
@@ -35,3 +35,63 @@ var validOverlayRequest = &cns.CreateNetworkContainerRequest{
 		},
 	},
 }
+
+var validVNETBlockRequest = &cns.CreateNetworkContainerRequest{
```

Comment:
Can you inline these vars in the body of the tests that use them or turn them into factory funcs?

Despite their prevalence in ACN code, using package variables for test data is not a best practice.